### PR TITLE
Change CTA label to delete selected entries

### LINKED ISSUE #3470: "Delete All" CTA message is unclear
**Describe the bug**
When selecting multiple entries in content manager (but not all entries) the multiple delete option says "Delete All"

**Expected behavior**
Option, provided all are not selected, should say "Delete Selected"

**System**
- Strapi version: v3.0.0-alpha.26.2

### PR DESCRIPTION
**Pull request by acupajoe**
#### Description of what you did:
Fixes: #3470 

Caveats:
- Missing translations
- "Delete All" only counts for what is on the current page as opposed to total records in the database. I've seen this rendered both ways out in the wild.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by acupajoe** [comment]
@derrickmehaffy would love if you could add translations for me!

**Comment by derrickmehaffy** [comment]
@acupajoe I just used google translate :P

**Comment by Aurelsicoko** [comment]
LGTM (still need the approval of @soupette)

File: packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js [hunk 1]
```diff
@@ -10,8 +10,9 @@ import { FormattedMessage } from 'react-intl';
 
 import styles from './styles.scss';
 
-function TableDelete({ colspan, number, onToggleDeleteAll }) {
+function TableDelete({ colspan, number, total, onToggleDeleteAll }) {
   const suffix = number > 1 ? 'plural' : 'singular';
+  const deleteMessageId = number === total ? 'delete' : 'deleteSelected';
```
Comment (alexandrebodin) [inline packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: Do we really consider it is a "delete all" if we only delete all visible items but we have more items in an other page.

e.g: select all 10 of first page among 100 pages does it really count as a "delete all" action ? IMO I don't think so

Comment (alexandrebodin) [reply packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: what do you think @soupette

Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: Well if you understand it as delete all item from this page if do believe that it is correct.

Comment (acupajoe) [reply packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: Is 'Delete all (on page)' a better alternative? Or is that too verbose?

Comment (alexandrebodin) [reply packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: too verbose I think. You can keep it like that If @soupette prefers

**Comment by acupajoe** [comment]
Additional translations have been added

File: packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js (hunk 1 above)
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js]: Yeah it's a bit too verbose.

**Pull request merged**